requirements.sh

```diff
@@ -1,17 +1,22 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
If so then we need to change throughout in the cibox *.sh files (provision, repository, somthing else)